packages/web-app-user-management/tests/unit/views/Users.spec.js

```diff
@@ -219,7 +218,7 @@ function getMountedWrapper({
 } = {}) {
   return shallowMount(Users, {
     localVue,
-    store: createStore(Vuex.Store, {
+    store: new Vuex.Store({
```

Comment:
We search and replaced all occurences of `vuex-extensions` `createStore` to be able to remove that dependency. 
Apart from that we did not touch this file, because it's changed in master and we didn't want to make our own lives even harder 😂

Reply:
.... relatable 😅 fine by me